src/BursarExportPlugin.tsx

```diff
@@ -0,0 +1,109 @@
+import {
+  Button,
+  LoadingPane,
+  Pane,
+  PaneFooter,
+} from '@folio/stripes/components';
+import { useStripes } from '@folio/stripes/core';
+import { FormApi } from 'final-form';
+import React, { useCallback, useRef } from 'react';
+import formValuesToDto from './api/dto/to/formValuesToDto';
+import schedulingToDto from './api/dto/to/schedulingToDto';
+import useAutomaticSchedulerMutation from './api/mutators/useAutomaticSchedulerMutation';
+import useManualSchedulerMutation from './api/mutators/useManualSchedulerMutation';
+import ConfigurationForm, { FORM_ID } from './form/ConfigurationForm';
+import useInitialValues from './hooks/useInitialValues';
+import FormValues from './types/FormValues';
+import { FormattedMessage } from 'react-intl';
+
+export default function BursarExportPlugin() {
+  const stripes = useStripes();
+
+  const manualScheduler = useManualSchedulerMutation();
```

Comment:
from naming it looks strange, when you call it `manualScheduler(formValues)` it doesn't sound like an actions

additionally when we see mutation hook, we expect it returns or mutator that has mutate function, or mutate function itself  (mutate can be any name `scheduleManually`, but it should be a function)